queue_entry.py

```diff
@@ -655,7 +653,8 @@ def execute(self):
         kappa = self._data_model.get_kappa()
         phi = self._data_model.get_kappa_phi()
 
-        self.diffractometer_hwobj.moveMotors({"kappa": kappa, "kappa_phi":phi})
+        if hasattr(self.diffractometer_hwobj, "in_kappa_mode") and self.diffractometer_hwobj.in_kappa_mode():
```

Comment:
Hi,

GenericDiffractometer has a method called `in_plate_mode()`. Could this be used instead of a `in_kappa_mode()`